Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_Internal_-_Generic_v2.yml

```diff
@@ -416,7 +437,16 @@ tasks:
                 iscontext: true
           accessor: Hostname
           transformers:
+          - operator: AppendIfNotEmpty
+            args:
+              item:
+                value:
+                  simple: inputs.Hostnames
+                iscontext: true
+              raw: {}
           - operator: uniq
+      UseReputationCommand:
+        simple: "False"
```

Comment:
When run from the parent playbook, it will be True. When run separately as standalone, it must run with False so that we don't break BC. The need to keep BC changed a lot of my decisions. I wanted to make a v3 but also wanted to improve this one.
Let me know if that makes sense

Reply:
Changed to True, pervious comment was referring to something else